addons/bus/static/src/js/services/async_service.js

```diff
@@ -0,0 +1,267 @@
+odoo.define('bus.AsyncJobService', function (require) {
```

Comment:
Considering name of file and JS module, this module should just return the service.
I don't agree with use of typedef for jobstate, but if you really want to keep it, should be defined in another file.